### ISSUE TITLE
Acari largely missing - consider miteresearch.org

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#410]
It seems COL is missing large parts of [Acari](https://www.catalogueoflife.org/data/taxon/LK). There are many families and superfamilies without any content, probably a relict from older times?

<img width="525" alt="image" src="https://user-images.githubusercontent.com/327505/160092778-6d01f597-f58d-4577-a947-9aa7f9adcbb1.png">

GBIF is in [touch](https://github.com/gbif/data-mobilization/issues/252) with http://www.miteresearch.org/ which might be a source to fill some gaps?

**Comment by DaveNicolson** [comment]
Acari is not a holophyletic group and is not used in modern classifications, but in any case, the gaps are known and actively being worked on through collaboration via the TG. Multiple mite GSDs have been added to COL in the last year, and several more are coming in the next ITIS load.

**Comment by DaveNicolson** [comment]
Superorders Acariformes and Parasitiformes are the bulk of the "mites", for the record

**Comment by mdoering** [comment]
Thats great. Maybe these are all small groups, but there are large numbers of empty superfamilies as you can see above (all the ones without any orange source name next to them have NO genera or species)

**Comment by DaveNicolson** [comment]
Yes, we intentionally added a global classification to family (published 2011-2013) that didn't include genera or species, which are to follow. GSDs are being sought and constructed. I'm noting this to Dr. Zhang, who is part of the effort and a mite worker. Since non-global and non-complete sources are no longer used in COL that does constrain sources somewhat.

**Comment by mdoering** [comment]
We should add all the non global parts in ITIS to the upcoming extended checklist.

**Comment by mdoering** [comment]
There are many Zootaxa family catalogues existing in Plazi that could be used by COL:

 - [Rhodacaridae 2012](https://tb.plazi.org/GgServer/summary/FFA24567FFF2FFADFFD7FF8FFFEF7936)
 - [Ascidae, Blattisociidae and Melicharidae 2016](https://tb.plazi.org/GgServer/summary/FFE9BF72FFB9D367C806663CC8379868)
 - [Pachylaelapidae 2014](https://tb.plazi.org/GgServer/summary/AE31321AFFB2FFB9FFB46A229C14FF97)
 - [Ologamasidae 2016](https://tb.plazi.org/GgServer/summary/FFD6650EC86F961BFFB8E76DFF980B38)
 - probably more...

**Comment by DaveNicolson** [comment]
Rhodacaridae gap filled in ITIS 3/2022
Pachylaelapidae gap filled in ITIS 3/2022
Most of the othes you mentioned are actively being worked on at ITIS now....
